rsmtool/container.py

```diff
@@ -260,21 +307,76 @@ def __iter__(self):
         for key in self.keys():
             yield key
 
+    def drop(self, name):
+        """
+        Drop a given data frame from the
+        container.
+
+        Parameters
+        ----------
+        name : str
+            The name of the data frame to drop from the
+            container object.
+
+        Returns
+        -------
+        self
+        """
+        if name in self._names:
+            self._names.remove(name)
+            self._dataframes.pop(name)
+            self._data_paths.pop(name)
+        else:
+            warnings.warn('The name `{}` is not in the '
+                          'container. No data frames will '
+                          'be dropped.'.format(name))
+        return self
+
+    def rename(self, name, new_name):
```

Comment:
Don't you want a warning here too? Basically don't we want a warning anytime we try to look for a frame that doesn't exist? Perhaps we could use an accessor method or something that has the warning?